src/definitions/definitionBlocks.ts

```diff
@@ -173,25 +159,20 @@ export class OutputDefinitionBlock<TypeName extends string> {
 
   protected decorateField(config: NexusOutputFieldDef): NexusOutputFieldDef {
     if (this.isList) {
-      if (config.list) {
+      if (isNexusListTypeDef(config.type)) {
```

Comment:
Should we be qualifying our internal function names like `isNexus...`?

Reply:
I think this is to differentiate from the `graphql-js` type defs